src/content/developers/tutorials/submitting-and-verifying-your-smart-contract-to-etherscan/index.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Submitting and Verifying your Smart Contract to Etherscan (Part 3/4 of Hello World dApp Series)
+description: Step by step guide on submitting your smart contract's ABI to Etherscan so anyone can look at the contract's methods and variables and understand how to interact with it.
+author: "nstrike2"
+tags:
+  [
+    "solidity",
+    "hardhat",
+    "truffle",
+    "alchemy",
+    "smart contracts",
+    "etherscan",
+    "getting started",
+    "verifying"
+  ]
+skill: beginner
+lang: en
+sidebar: true
+published: 2021-10-06
+---
+
+# Submitting your Smart Contract to Etherscan
+
+You should have completed part 1 [creating and deploying a smart contract](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract), and part 2 [interacting with your smart contract](https://docs.alchemy.com/alchemy/tutorials/hello-world-smart-contract/interacting-with-a-smart-contract) prior to starting part 3 below.
```

Comment:
Note: we'll should probably change this to the ethereum.org version once this is live